### DISCUSSION TITLE
On the Project / Board view Milestone should be visible if set

### DISCUSSION THREAD

**Discussion by kozaka-tv** [community/community#33483] [Projects and Issues] [answered]
I set milestones to some issues, but this is not visible on board view of a project. It would be great if we could have this information on the board:

![image](https://user-images.githubusercontent.com/4964462/190991640-f1b49fe3-3035-48b0-bd1e-dd5a077bda06.png)

**Comment by timoyates** [comment, marked as answer]
Hi @kozaka-tv - if you click that little down arrow on the view tab next to the view's title, you can toggle which fields are visible and invisible (don't forget to save your changes to those settings, though!)

**Comment by kozaka-tv** [comment]
Well! That helped, thank you!
